plugins/parsers/grok/parser_test.go

```diff
@@ -0,0 +1,19 @@
+package grok
```

Comment:
Needs more tests, probably easiest to just bring over all tests from grok_test.go